tests/testsuite/cargo_information/pick_msrv_compatible_package_within_ws_and_use_msrv_from_root_manifest/in/crate1/Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+name = "crate1"
+version = "0.0.0"
+
```

Comment:
I'm not seeing how the test from this commit and the previous are significantly different.

The only difference I observe is that `crate1` doesn't have a rust-version but that doesn't seem relevant and is independent of "from_root_manifest".

Reply:
I renamed it. The point of this case is we should pick the MSRV from crate2.

Reply:
See: https://github.com/hi-rustin/cargo-information/pull/74#discussion_r1435771876

Reply:
Thanks for the explanation!